content/html-css/sprints/1/day-plan/index.md

```diff
@@ -7,6 +7,19 @@ weight = 3
 backlog= 'Module-HTML-CSS'
 backlog_filter= 'Week 1'
 [[blocks]]
+name="energiser"
+[[blocks]]
+name="Devtools Workshop"
```

Comment:
This seems to be rendering front-matter?

<img width="542" alt="Screenshot 2023-07-23 at 21 50 18" src="https://github.com/CodeYourFuture/curriculum/assets/1131704/101d72bf-7eae-4964-954f-3c3bb5384943">

Reply:
Oh hmmmm. Let me update this PR from main and see if that's still happening. I wanted to parse front matter but couldn't get it working so I thought I had turned it off.